lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -602,6 +612,17 @@ def from_ansible_params(ap, old_service):
         s.update_order = ap['update_order']
         s.user = ap['user']
 
+        s.command = ap['command']
+        if isinstance(s.command, string_types):
+            s.command = shlex.split(s.command)
+        elif isinstance(s.command, list):
+            s.command = ap['command']
```

Comment:
You might also want to check the elements of the list here. If they aren't strings, you should fail (or convert them to string and output a warning) and tell the user to quote the values properly.